shoebot/data/typography.py

```diff
@@ -112,7 +112,8 @@ class Text(Grob, ColorMixin):
         Implementation of fonts uses Pango instead of Cocoa.
     """
 
-    # several reference docs can be found at http://www.pyGtk.org/docs/pygtk/class-pangofontdescription.html
+    # several reference docs can be found at
+    # https://developer.gnome.org/pygtk/stable/class-pangofontdescription.html
```

Comment:
Is this right ?  Pygtk was the Gtk2 thing.

Reply:
Ah you're right! Still the API is pretty much the same, but in a quick search i couldn't find the current pango docs

Reply:
I'll just remove the comment

Reply:
Yeah I couldn't find the current docs either, I guess either remove it or mention it.